web/cobrands/sass/_base.scss

```diff
@@ -1656,7 +1699,7 @@ a:hover.rap-notes-trigger {
 // and make the map full screen to reduce distractions. JavaScript also
 // tweaks the text content of some of the map-related elements, to make
 // it more "app-like".
-.mobile-reporting-map {
+.mobile-reporting-map, .report-fullscreen {
```

Comment:
Yeah, I like this, then if we want to use `map-fullscreen` anywhere else, it makes sense, without having to add another class